File: Modules/School/Question/QuestionForm.cs
using Serenity.ComponentModel;
using System;

namespace ExamAppSerene.School.Forms;

[FormScript("School.Question")]
[BasedOnRow(typeof(QuestionRow), CheckNames = true)]
public class QuestionForm
{
    public string QuestionTitle { get; set; }
    public string OptionA { get; set; }
    public string OptionB { get; set; }
    public string OptionC { get; set; }
    public string OptionD { get; set; }
    public int AnswerKey { get; set; }
    public bool Status { get; set; }
    public DateTime CreatedDate { get; set; }
    public int ExamId { get; set; }
}

File: Modules/ServerTypes/School/QuestionForm.ts
import { StringEditor, IntegerEditor, BooleanEditor, DateEditor, LookupEditor, PrefixedContext } from "@serenity-is/corelib";
import { initFormType } from "@serenity-is/corelib/q";

export interface QuestionForm {
    QuestionTitle: StringEditor;
    OptionA: StringEditor;
    OptionB: StringEditor;
    OptionC: StringEditor;
    OptionD: StringEditor;
    AnswerKey: IntegerEditor;
    Status: BooleanEditor;
    CreatedDate: DateEditor;
    ExamId: LookupEditor;
}

export class QuestionForm extends PrefixedContext {
    static formKey = 'School.Question';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!QuestionForm.init)  {
            QuestionForm.init = true;

            var w0 = StringEditor;
            var w1 = IntegerEditor;
            var w2 = BooleanEditor;
            var w3 = DateEditor;
            var w4 = LookupEditor;

            initFormType(QuestionForm, [
                'QuestionTitle', w0,
                'OptionA', w0,
                'OptionB', w0,
                'OptionC', w0,
                'OptionD', w0,
                'AnswerKey', w1,
                'Status', w2,
                'CreatedDate', w3,
                'ExamId', w4
            ]);
        }
    }
}